src/assets/reporting_parent_sites.svg

```diff
@@ -0,0 +1 @@
" style="font-family: 'Open Sans', verdana, arial, sans-serif; font-size: 12px; fill: rgb(68, 68, 68); fill-opacity: 1; visibility: visible; white-space: pre;">100</text></g></g><g class="overaxes"/></g></g><g class="ternarylayer"/><g class="geolayer"/><g class="layer-above"><g class="imagelayer"/><g class="shapelayer"/></g><g class="pielayer"/><g class="glimages"/><defs id="topdefs-b2d7ff"><clipPath id="legendb2d7ff"><rect width="515" height="32" x="0" y="0"/></clipPath></defs><g class="infolayer"><g class="g-gtitle"/><g class="g-xtitle"/><g class="g-ytitle" data-bb="68"><text class="ytitle" transform="rotate(-90,32,185.5) translate(0, 0)" x="32" y="185.5" text-anchor="middle" style="font-family: 'Open Sans', verdana, arial, sans-serif; font-size: 14px; fill: rgb(68, 68, 68); opacity: 1; font-weight: normal; white-space: pre; visibility: visible;">%</text></g><g class="legend" pointer-events="all" transform="translate(285, 12.099999999999973)"><rect class="bg" shape-rendering="crispEdges" style="stroke: rgb(68, 68, 68); stroke-opacity: 1; fill: rgb(255, 255, 255); fill-opacity: 1; stroke-width: 0px;" width="515" height="32" x="0" y="0"/><g class="scrollbox" transform="translate(0, 0)" clip-path="url(#legendb2d7ff)"><g class="groups"><g class="traces" transform="translate(0, 15.6)" style="opacity: 1;"><g class="layers" style="opacity: 1;"><g class="legendfill"/><g class="legendlines"/><g class="legendsymbols"><g class="legendpoints"><path class="legendbar" d="M6,6H-6V-6H6Z" transform="translate(20,0)" style="stroke-width: 0px; fill: rgb(90, 10, 105); fill-opacity: 1;"/></g></g></g><text class="legendtext user-select-none" x="40" y="5.46" style="text-anchor: start; font-family: 'Open Sans', verdana, arial, sans-serif; font-size: 14px; fill: rgb(68, 68, 68); fill-opacity: 1; white-space: pre; visibility: visible;" data-bb="75">Completeness of weekly reporting</text><rect class="legendtoggle" pointer-events="all" style="cursor: pointer; fill: rgb(0, 0, 0); fill-opacity: 0;" x="0" y="-10.6" width="515" height="21.2"/></g><g class="traces" transform="translate(257.06231689453125, 15.6)" style="opacity: 1;"><g class="layers" style="opacity: 1;"><g class="legendfill"/><g class="legendlines"/><g class="legendsymbols"><g class="legendpoints"><path class="legendbar" d="M6,6H-6V-6H6Z" transform="translate(20,0)" style="stroke-width: 0px; fill: rgb(98, 178, 0); fill-opacity: 1;"/></g></g></g><text class="legendtext user-select-none" x="40" y="5.46" style="text-anchor: start; font-family: 'Open Sans', verdana, arial, sans-serif; font-size: 14px; fill: rgb(68, 68, 68); fill-opacity: 1; white-space: pre; visibility: visible;" data-bb="76">Timeliness of weekly reporting</text><rect class="legendtoggle" pointer-events="all" x="0" y="-10.6" width="515" height="21.2" style="cursor: pointer; fill: rgb(0, 0, 0); fill-opacity: 0;"/></g></g></g><rect class="scrollbar" rx="20" ry="2" width="0" height="0" style="fill: rgb(128, 139, 164); fill-opacity: 1;"/></g></g></svg>
```

Comment:
hmm I'm still not sure that we should include these files in the repo? As far as I understand, these plots will be regenerated each time we have new data, is that correct? In this case it feels weird to have one certain version in the repo.

What would be needed so that we don't have to add them?